packages/framework/src/Form/Admin/Product/Brand/BrandFormType.php

```diff
@@ -80,22 +82,68 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ];
         }
 
-        $builder
+        $builderBasicInformationGroup = $builder->create('basicInformation', GroupType::class, [
+            'label' => t('Basic information'),
+        ]);
+        $builderBasicInformationGroup
+            ->add('id', DisplayOnlyType::class, [
+                'label' => t('ID'),
+                'data' => $brand !== null ? $brand->getId() : '',
+            ])
             ->add('name', TextType::class, [
                 'required' => true,
                 'constraints' => [
                     new Constraints\NotBlank(['message' => 'Please enter name']),
                     new Constraints\Length(['max' => 255, 'maxMessage' => 'Name cannot be longer than {{ limit }} characters']),
                 ],
+                'label' => t('Name')
             ])
             ->add('descriptions', LocalizedType::class, [
                 'entry_type' => CKEditorType::class,
                 'required' => false,
+                'label' => t('Description')
+            ]);
+
+        $builderSeoGroup = $builder->create('seo', GroupType::class, [
+            'label' => t('Seo'),
+        ]);
+        $builderSeoGroup
+            ->add('seoTitles', MultidomainType::class, [
+                'entry_type' => TextType::class,
+                'required' => false,
+                'options_by_domain_id' => $seoTitlesOptionsByDomainId,
+                'macro' => [
+                    'name' => 'seoFormRowMacros.multidomainRow',
+                    'recommended_length' => 60,
+                ],
+                'label' => t('Page title')
+            ])
+            ->add('seoMetaDescriptions', MultidomainType::class, [
+                'entry_type' => TextareaType::class,
+                'required' => false,
+                'options_by_domain_id' => $seoMetaDescriptionsOptionsByDomainId,
+                'macro' => [
+                    'name' => 'seoFormRowMacros.multidomainRow',
+                    'recommended_length' => 155,
+                ],
+                'label' => t('Meta description'),
+            ])
+            ->add('seoH1s', MultidomainType::class, [
+                'entry_type' => TextType::class,
+                'required' => false,
+                'options_by_domain_id' => $seoH1sOptionsByDomainId,
+                'label' => t('Heading (H1)')
             ])
             ->add('urls', UrlListType::class, [
```

Comment:
The `urls` shouldn't be a part of the form at all when a new Brand is created (i.e. when `$options['brand']` is `null`).

See the new Brand page on *master*.

Reply:
Look at the ArticleFormType for inspiration: https://github.com/shopsys/shopsys/blob/master/packages/framework/src/Form/Admin/Article/ArticleFormType.php#L143-L150